### DISCUSSION TITLE
Auto-conversion modification?

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#4094] [Q&A]
Where can I find the auto-conversion code used by mv? The idea is to tweak the code so i get slightly lighter models on iOS after auto-conversion is done.

**Comment by elalish** [comment]
We use this: https://github.com/mrdoob/three.js/pull/24950#issuecomment-1313842312

I believe it always adds a specular-extension texture, but in practice this often makes very little difference. You might try removing that extension and see how the result looks.

**Reply by danilokrulj** [reply]
That is the usdz auto-conversion code, right? Just want to confirm

**Reply by elalish** [reply]
No, it's not - did you follow the link?

**Reply by danilokrulj** [reply]
I did and I was looking for the usdz conversion but i couldn't find it anywhere.

**Reply by elalish** [reply]
Oh, sorry - the auto-conversion we just added was specGloss->metalRough so I thought that's what you were referring to. For USDZ we just use the three.js USDZExporter.
